test/Website/Program.cs

```diff
@@ -7,7 +7,7 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
+            IWebHost host = new WebHostBuilder()
```

Comment:
Sorry about this, I don't know why R# didn't catch this file in the previous code cleanup